troposphere/rds.py

```diff
@@ -137,7 +137,7 @@ class DBInstance(AWSObject):
         'DBSecurityGroups': (list, False),
         'DBSnapshotIdentifier': (basestring, False),
         'DBSubnetGroupName': (basestring, False),
-        'Engine': (validate_engine, True),
```

Comment:
The use of validate_engine can still be used here instead of adding additional code in validate but change required to False.